src/components/pagination/pagination.e2e.ts

```diff
@@ -179,4 +180,70 @@ describe("calcite-pagination", () => {
       expect(toggleSpy).toHaveReceivedEventTimes(0);
     });
   });
+
+  describe("number locale support", () => {
+    let page: E2EPage;
+    let element: E2EElement;
+    let noSeparator: string[];
```

Comment:
This and the variable below can be moved inside their respective functions. Reasoning: they are not initialized in `beforeEach` and are used in different contexts (not shared across tests).